Support Prisma 6

Hi, thanks for this great extension!

Could we please add support for the newly released [Prisma 6](https://github.com/prisma/prisma/releases/tag/6.0.0)? Shouldn't have any breaking but right now, if I try to upgrade Prisma, I get this:

> ❯ npm i
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
...
npm error Could not resolve dependency:
npm error peer @prisma/client@"^4.9.0 || ^5.0.0" from prisma-extension-pagination@0.7.4
npm error node_modules/prisma-extension-pagination
npm error   prisma-extension-pagination@"^0.7.4" from the root project

It should be enough to upgrade [this project peer dependencies](https://github.com/deptyped/prisma-extension-pagination/blob/main/package.json#L36)

--- a/package.json
+++ b/package.json
@@ -36,6 +36,7 @@
     "@prisma/client": "^4.9.0 || ^5.0.0 || ^6.0.0"
   },
   "devDependencies": {
+    "@prisma/client": "^6.0.1",
     "@types/jest": "^29.5.14",
     "@types/node": "^20.12.7",
     "@typescript-eslint/eslint-plugin": "^7.6.0",

--- a/package.json
+++ b/package.json
@@ -33,19 +33,19 @@
   },
   "homepage": "https://github.com/deptyped/prisma-extension-pagination#readme",
   "peerDependencies": {
-    "@prisma/client": "^4.9.0 || ^5.0.0"
+    "@prisma/client": "^4.9.0 || ^5.0.0 || ^6.0.0"
   },
   "devDependencies": {
-    "@types/jest": "^29.5.12",
+    "@types/jest": "^29.5.14",
     "@types/node": "^20.12.7",
     "@typescript-eslint/eslint-plugin": "^7.6.0",
     "@typescript-eslint/parser": "^7.6.0",
     "eslint": "^8.55.0",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-prettier": "^5.2.1",
     "jest": "^29.7.0",
-    "prisma": "^5.7.0",
-    "ts-jest": "^29.1.2",
-    "typescript": "^5.4.5"
+    "prisma": "^6.0.1",
+    "ts-jest": "^29.2.5",
+    "typescript": "^5.7.2"
   }
 }

